gradle/check-licenses.gradle

```diff
@@ -30,17 +30,21 @@
 ext.acceptedLicenses = [
         'BSD License',
         'BSD 3-Clause',
+        'COMMON DEVELOPMENT AND DISTRIBUTION LICENSE (CDDL) Version 1.0',
         'Eclipse Public License - v 1.0',
         'Eclipse Public License - v 2.0',
         'Eclipse Public License 2.0',
         'Eclipse Public License 1.0',
+        'GPLv2+CE',
```

Comment:
Is this addition really needed?

Reply:
Azure depends on javax:mail which is GPLv2 OR CDDL - so have updated appropriately.